### PR TITLE
kubeadm: fix reset error logging.

### PR DESCRIPTION
**Pull request by pires**
**What this PR does / why we need it**: while investigating https://github.com/kubernetes/kubeadm/issues/142 I realized error logging was misleading. This PR is meant to fix it.

**Special notes for your reviewer**: /cc @luxas 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/reset.go [hunk 1]
```diff
@@ -144,36 +144,39 @@ func (r *Reset) Run(out io.Writer) error {
...
 	if err != nil {
 		return fmt.Errorf("failed to detect node hostname")
 	}
 	hostname = strings.ToLower(hostname)
 
 	// TODO: Use the "native" k8s client for this once we're confident the versioned is working
-	kubeConfigPath := filepath.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, kubeconfig.KubeletKubeConfigFileName)
+	kubeConfigPath := filepath.Join(kubeadmapi.GlobalEnvParams.KubernetesDir, kubeconfig.AdminKubeConfigFileName)
+
+	fmt.Println(kubeConfigPath)
```
Comment (luxas) [inline cmd/kubeadm/app/cmd/reset.go]: Why print this here? I guess it was included in the commit by accident

**Review by luxas — COMMENTED** [review]
LGTM except for one nit

File: cmd/kubeadm/app/cmd/reset.go (hunk 1 above)
Comment (pires) [inline cmd/kubeadm/app/cmd/reset.go]: Yes, it was.

**Comment by pires** [comment]
Nice catch @luxas. A `git commit -a --amend` had screwed the initial commit. Fixed and pushed.

**Comment by pires** [comment]
@k8s-bot cvm gke e2e test this

**Comment by pires** [comment]
@k8s-bot kops aws e2e test this

**Comment by luxas** [comment]
@k8s-bot cvm gke e2e test this
@k8s-bot kops aws e2e test this

**Pull request merged**